wiremock/assessmentStubs.ts

```diff
@@ -86,6 +87,19 @@ const assessmentStubs = [
             jsonBody: {},
           },
         },
+        {
+          priority: 99,
```

Comment:
Generally when we're stubbing a POST endpoint, we'll have the success response that matches against a path, and some error responses that match against the path plus specific fields being missing. We want to make sure that the success response takes lower priority, otherwise that'd match first and we'd never see the error response in our wiremock API

Worth having a look at `errorStub()` and `getCombinations()`. We don't use the latter here as there's only one field (`message`), but generally we use the two together to generate error stubs for any combination of missing fields